AppInspector/rules/default/os/system_registry.json

```diff
@@ -190,23 +226,39 @@
   },
   {
     "name": "OS Operation: Windows Registry (Read)",
-    "id": "AI038600",
+    "id": "AI038601",
     "description": "OS Operation: Windows Registry (Read)",
     "tags": [
       "OS.SystemRegistry.Read"
     ],
     "applies_to": [
-      "csharp"
+      "rust"
     ],
     "severity": "moderate",
     "patterns": [
       {
-        "pattern": "Registry(Key)?(.*)?.GetValue",
-        "type": "regex",
+        "pattern": ".open_subkey(",
+        "type": "string",
         "scopes": [ "code" ],
         "confidence": "high",
         "_comment": ""
       }
+    ],
+    "conditions": [
+      {
+        "pattern": {
+          "pattern": "HKEY_",
+          "type": "regex-word",
+          "scopes": [
+            "code"
+          ],
+          "modifiers": [ "i" ],
```

Comment:
I thought you changed this to be case insensitive

Reply:
I did and missed one in a condition section at the end.  Fixing now.

Reply:
Reviewed the file and others have exact api names and no need for it there also.  I had focused on the comment earlier for HKEY too narrowly but have removed then elsewhere also where not needed in that file.  Often a copy paste to add a new rule etc.  Sometimes I don't notice the i.